src/constants/indicators.js

```diff
@@ -10,6 +10,7 @@ export default {
     longDescription: 'This shows the population size.\n\nSource: ABS - Regional Population Growth',
     format: '0[.]0a',
     contextual: true,
+    lastUpdated: '2016',
```

Comment:
The designs show a full date, e.g. `27 July 2016`, rather than just the year. Was that data not available?